wrappers/rust/icicle-core/src/ecntt/mod.rs

```diff
@@ -161,6 +161,7 @@ macro_rules! impl_ecntt_bench {
         use icicle_core::ntt::ntt;
         use icicle_core::ntt::NTTDomain;
         use icicle_cuda_runtime::memory::HostOrDeviceSlice;
+        use std::env;
```

Comment:
what's that for?

Reply:
to pass external max log2 size of test, instead of hardcode